### PR TITLE
examples： fix udp-client

### PR DESCRIPTION
**Pull request by nkbai**
## Motivation
  use std::io::stdin.read_to_end will block the task.
    it's not a good example,

## Solution
  use tokio::io::stdin instead.

### CODE REVIEW

**Comment by LucioFranco** [comment]
looks like your branch is out of date with master, I'd recommend rebasing.

**Comment by carllerche** [comment]
Yes, the change looks good, but something wonky is going on with the history.

**Comment by nkbai** [comment]
@LucioFranco  thanks, I have already rebased.

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 target
 Cargo.lock
+.idea
```
Comment (LucioFranco) [inline .gitignore]: Do you mind removing this? I'd like to keep editor specific ignores out of here, you can use a global gitignore for this.

File: tokio-tls/examples/echo.rs [hunk 2]
```diff
@@ -0,0 +1,55 @@
+#![warn(rust_2018_idioms)]
```
Comment (LucioFranco) [inline tokio-tls/examples/echo.rs]: So it looks like this got renamed to https://github.com/tokio-rs/tokio/blob/master/tokio-tls/examples/tls-echo.rs

So I don't think we should add this here.

Comment (nkbai) [reply tokio-tls/examples/echo.rs]: it seems that every file of tokio has this line

**Comment by Darksonn** [comment]
The history needs to be fixed if this is to be merged.

**Pull request closed without merging**